joomla/joomla-cms
# [4.0] Remove remaining 'column #' options from blog/featured

Pull Request for Issue #21106 .

### Summary of Changes
This field no longer has any effect. This PR removes the remaining traces of it from com_content featured/blog. Column count can now be set by https://github.com/joomla/joomla-cms/pull/18319


### Testing Instructions
Apply PR and check that **# Columns** field has been removed from the 'Featured Articles' menu item.


### Expected result



### Actual result



### Documentation Changes Required
Yes
